src/test/resources/testdata/licence-condition.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
I think it would be better to put these in a folder called licence-condition as we're going to end with loads of these for every endpoint